pom.xml

```diff
@@ -657,6 +657,116 @@
             </build>
         </profile>
 
+        <profile>
+            <id>payara-micro-remote</id>
+
+            <properties>
+                <!-- default port -->
+                <payara.httpPort>8080</payara.httpPort>
```

Comment:
Does it support parallel test execution ?

Reply:
Well since deployments can be done in parallel, I don't see why it wouldn't support parallel execution.

Reply:
At least that's as far as theory goes.
Inpractice,  `mvn -T2` fails with classloading errors, just like `-Dparallel=classes` does in projects with multiple test classes. So I must correct the answer to "no, it doesn't".

`-Dparallel=methods` work, though (but may not play well with arquillians `@InOrder` annotation)

Do other profiles support parallel executions?